cockatrice/src/pictureloader.cpp

```diff
@@ -121,10 +95,8 @@ QString PictureToLoad::getSetName() const
     }
 }
 
-QStringList PictureLoaderWorker::md5Blacklist = QStringList()
-                                                << "db0c48db407a907c16ade38de048a441"; // card back returned
-                                                                                       // by gatherer when
-                                                                                       // card is not found
+// Card back returned by gatherer when card is not found
```

Comment:
How does scryfall behave here? I'm presuming it gives a 404 instead of the cardback?

Reply:
That would be correct. This is kept here for Gatherer issues.